docs/interaction_code_flow.md

```diff
@@ -619,7 +619,7 @@ new OktaSignIn({
 
 // forgot_password.html
```

Comment:
nit: should we change this to `reset_password.html` since the flow name has changed?